enhancements/proposals/disk_setup.md

```diff
@@ -0,0 +1,225 @@
+    PVProvisionerDisabled bool `json:"pvProvisionerDisabled"`
+    
+    // pvProvisionerOptions contains options for deployed PV Provisioner.
+    PVProvisionerOptions *DiskProvisionerOptions `json:"pvProvisionerOptions"`
+
+    // diskBenchmarkDisabled controls if disk benchmark cacheing should be disabled.
+    DiskBenchmarkDisabled bool `json:"diskBenchmarkDisabled"`
+    // diskBenchmarkOptions contains options for disk benchmark cacheing.
+    DiskBenchmarkOptions  *ScyllaNodeTunerDiskBenchmarkOptions `json:"diskBenchmarkOptions"`
+}
+
+```
+
+### RBAC
+
+ClusterRole bound to ServiceAccount used by ScyllaNodeTuner DaemonSet is going to be extended with read permissions to PersistentVolumes and Pods.
+
+Operator will provision a new ServiceAccount for Disk Provisioner DaemonSet with `system:persistent-volume-provisioner` ClusterRole.
+
+### Flow
+
+Scylla Operator is going to reconcile Disk Provisioner and ScyllaNodeTuner DaemonSet based on values in available ScyllaNodeTuner CR.
+
+If `ScyllaNodeTuner.DiskSetupDisabled` is `false` ScyllaNodeTuner DaemonSet is going to run `scylla_raid_setup` script which 
+discovers disks attached to the instance and creates RAID0, formats it to XFS. Created disks will be mounted in 
+location specified in `ScyllaNodeTuner.LocalDisksPath`.
+
+When new disk appears in specified location, Disk Provisioner will expose them as PersistentVolumes.
+
+Sidecar before Scylla process is spawned, checks if ConfigMap `scylla-node-tuner` mounted as directory contains `iosetup-<pv-uid>` file. 
+It doesn't use API, but check if filesystem contains such file.
+If it exists and is not empty, its content is copied to `/etc/scylla.d/io_properties.yaml`. If not, sidecar proceeds to spawning Scylla.
+If `ScyllaNodeTuner.DiskBenchmarkDisabled` is `true` Operator creates an empty `iosetup-<pv-uid>` key in `scylla-node-tuner` ConfigMap to unblock sidecar.
+Otherwise, ScyllaNodeTuner is responsible for filling it in.
+
+### Precomputed benchmark
+
+`scylla_io_setup` script which contains logic for detecting and calculating precomputed values for known instances requires
+special parameter (--ami) in order to skip actual benchmark on AWS and to use precomputed values. 
+This parameter has unfortunate behavior when AWS instances are *not* in recommended set (i3, i3en, and more) - it just exits with 1 code.  
+So before running this script with this parameter we have to be sure that we are on AWS and instance is recommended.  
+
+ScyllaNodeTuner Pod is going to detect if we are running on AWS, and if instance type is one of the recommened types 
+(we will hardcode the list based on `scylla_util`). If so an `--ami` parameter is added to `scylla_io_setup`.
+GKE and other Clouds and deployments doesn't require any additional flags.  
+
+If `ScyllaNodeTuner.DiskBenchmarkOptions.IgnorePrecomputed` is `true`, instead running `scylla_io_setup` ScyllaNodeTuner will direcly execute `iotune`.
```

Comment:
```suggestion
If `ScyllaNodeTuner.DiskBenchmarkOptions.IgnorePrecomputed` is `true`, instead of running `scylla_io_setup` ScyllaNodeTuner will direcly execute `iotune`.
```